sound/soc/sof/intel/hda-stream.c

```diff
@@ -433,15 +433,23 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 irqreturn_t hda_dsp_stream_interrupt(int irq, void *context)
 {
 	struct hdac_bus *bus = context;
+	struct sof_intel_hda_dev *sof_hda = bus_to_sof_hda(bus);
+	u32 stream_mask;
 	u32 status;
 
+	dev_vdbg(bus->dev, "stream irq, INTSTS status: 0x%x\n",
+		 snd_hdac_chip_readl(bus, INTSTS));
+
```

Comment:
does it makes sense to place the log here, knowing that you read the same value below inside of a spin_lock?
Either remove this log or do this right.

Reply:
@plbossart I'd like to keep this here, as we need debug return before the spin_lock below sometimes (returned at pm_runtime_active failed).

Reply:
then add a comment that this is explicit and not a mistake